client/src/stories/1-Input.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { LoginInput } from '../components/Input';
+
+export default {
+  title: 'LoginInput',
+};
+
+export const MailLogin = (props) => (
```

Comment:
I don't think you will have props here in the stories. You can safely remove `props` from the stories.